service/installer.sh

```diff
@@ -37,12 +43,12 @@ function fetchBlob(){
 	filemod=$3
 
 	source="$RAW_REPO/$LATEST_TAG/$file"
-	echo "==> Downloading $source"
+	echo -e "\e[34m==> Downloading $source\e[0m"
 	tmpf=$(mktemp -t go-openbmclapi.XXXXXXXXXXXX.downloading)
 	curl -fsSL -o "$tmpf" "$source" || { rm "$tmpf"; return 1; }
-	echo "==> Downloaded $source"
+	echo -e "\e[34m==> Downloaded $source\e[0m"
 	mv "$tmpf" "$target" || return $?
-	echo "==> Installed to $target"
+	echo -e "\e[34m==> Installed to $target\e[0m"
 	if [ -n "$filemod" ]; then
```

Comment:
```suggestion
	chown $USERNAME "$target"
	if [ -n "$filemod" ]; then
```
因为换了个用户，所以得加个`chown`